src/api/cvc4cpp.cpp

```diff
@@ -5678,6 +5678,74 @@ SmtEngine* Solver::getSmtEngine(void) const { return d_smtEngine.get(); }
  */
 Options& Solver::getOptions(void) { return d_smtEngine->getOptions(); }
 
+/* -------------------------------------------------------------------------- */
```

Comment:
I don't know, this seems too long :) even though i see it is 80 chars.
Maybe check how such comments look around other functions in this file (or other files).